munich/README

```diff
@@ -1,7 +1,19 @@
 ###PyLadies Munich
 
-The repo for PyLadies Munich. 
+We are the Munich chapter of an international mentorship group, PyLadies, with a focus on helping more women become active participants and leaders in the Python open-source community. Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
 
-Get social with us! Twitter, via [@pyladiesmunich][1]!
+PyLadies also aims to provide a friendly support network for women and a bridge to the larger Python world. Anyone with an interest in Python is encouraged to participate!
 
-[1]: http://www.twitter.com/pyladiesmunich
+Code of Conduct: https://www.pyladies.com/CodeOfConduct/
+
+http://www.pyladies.com | http://munich.pyladies.com
+
+Follow us:
+MeetUp [@pyladiesmunich][1]!
```

Comment:
I would prefix the Meetup and Facebook lines with `- `. Otherwise will run the lines together

Reply:
I agree :)